charts/cryostat/templates/persistent-volumelaim.yaml

```diff
@@ -0,0 +1,10 @@
+{{- if .Values.pvc.enabled }}
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: {{ .Release.Name }}-persistent-volumeclaim
```

Comment:
We could include the labels here used by other chart components. It should be as simple as adding these two lines:
https://github.com/cryostatio/cryostat-helm/blob/541adadb520f4d9d534d9415d2471ee8e3fe20fa/charts/cryostat/templates/ingress.yaml#L22-L23